src/handlers/interventions-processor-handler.ts

```diff
@@ -135,12 +140,13 @@ async function handleError(error: unknown, record: SQSRecord) {
     });
   else if (error instanceof StateTransitionError) {
     logger.warn('StateTransitionError caught, message will not be retried.', { errorMessage: error.message });
-    await sendAuditEvent(
-      'AIS_EVENT_TRANSITION_IGNORED',
+    const recordBody = JSON.parse(record.body) as TxMAIngressEvent;
+    await new AuditEvents(
+      TxMAEgressEventTransitionType.TRANSITION_IGNORED,
```

Comment:
are we changing the event names from 'AIS_EVENT_TRANSITION_IGNORED' to just 'TRANSITION_IGNORED'?

Reply:
We're only changing it here, not on the message to send to txma.
Before sending it to txma it is manipulated according to the ingress event source (fraud/ non fraud)
see `extractEgressEventName` in AuditEvents